### DISCUSSION TITLE
Changing Teleport default SSH Port

### DISCUSSION THREAD

**Discussion by Reab0** [gravitational/teleport#7727] [Q&A] [answered]
Hi, 
I would like to register a node in the cluster. However this node has a service that listens to port 3022 for ssh auth. Can I also start teleport on that node using the same port 3022 for ssh?
If not how can I change teleport default SSH port in both proxy and node services to a different value? 

Thanks,

**Comment by Reab0** [comment, marked as answer]
Port changed by starting the node using config file.

**Comment by webvictim** [comment]
You can change the listen address for the SSH service and restart Teleport:
```yaml
ssh_service:
  listen_addr: 0.0.0.0:4022
```
